libursa/src/utils/macros.rs

```diff
@@ -17,6 +17,20 @@ macro_rules! array_copy {
     };
 }
 
+#[cfg(any(
```

Comment:
Wouldn't this feature set work for only `impl_bytearray`? You need this feature set for `secret` macro.

Reply:
Yes, that's the idea. `secret` isn't gated by additional features, only the features on the whole module.

Reply:
I misunderstood. I thought you were trying to make `secret!` gated.